test/replutil.jl

```diff
@@ -252,12 +253,6 @@ let
     @test contains(err_str, "column vector")
 end
 
-abstract type T11007 end
-let
-    err_str = @except_str T11007() MethodError
-    @test contains(err_str, "no method matching $(curmod_prefix)T11007()")
```

Comment:
I added a test that covers this case earlier; my test also checks that the message reads 
`no method matching` when there is at least one constructor defined.